cmd/root.go

```diff
@@ -63,8 +66,14 @@ func Execute() {
 }
 
 func init() {
-	// TODO - read from /etc/casaos/gateway.ini
-	rootCmd.PersistentFlags().StringP(FlagRootURL, "u", "localhost:80", "root url of CasaOS API")
+	cfgs, err := ini.Load("/etc/casaos/gateway.ini")
```

Comment:
better be an argument with a default value `/etc/casaos/gateway.ini` - try not to make it a hardcoded string.